cmd/buildah/images_test.go

```diff
@@ -50,6 +50,23 @@ func TestTemplateOutputValidTemplate(t *testing.T) {
 	}
 }
 
+func TestTemplateOutputInvalidFormat(t *testing.T) {
```

Comment:
I think you're having a collision with this function name and the one in the containers test

Reply:
updated, PTAL.